docs/data-ai/ai/train.md

```diff
@@ -90,15 +88,17 @@ If you haven't already, create a folder called <file>model</file> and create an
 
 <p><strong>4. Add <code>training.py</code> code</strong></p>
 
-<p>Copy this template into <file>training.py</file>:</p>
+<p>Copy one of the following templates into <file>training.py</file>, depending on whether or not you wish to parse labels:</p>
```

Comment:
Can you rephrase this to have the condition based on which the reader should decide be clearer? Why would a user need to use the labels flag or not?